app/src/main/java/com/example/mymovie/FragmentMoviesDetails.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.mymovie
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import org.w3c.dom.Text
+
+//  inflate fragment_movies_details.xml in it
+
+class FragmentMoviesDetails : Fragment() {
```

Comment:
lifehack:
https://medium.com/android-news/define-resource-layout-id-in-the-constructor-activity-fragment-with-androidx-a9f80674f185